### PR TITLE
18.2.1 Release notes

### PR DESCRIPTION
**Pull request by yuriw**

### CODE REVIEW

**Comment by yuriw** [comment]
https://wip-yuriw-release-18.2.1-post-main.ceph.io/en/news/blog/2023/v18-2-1-reef-released/

File: src/en/news/blog/2023/v18-2-1-reef-released/index.md [hunk 1]
```diff
@@ -0,0 +1,891 @@
+---
+title: "v18.2.1 Reef released"
+date: "2023-11-20"
+author: "Yuri Weinstein"
+tags:
+  - "release"
+  - "reef"
+---
+
+This is the first backport release in the Reef series. We recommend all users update to this release.
```
Comment (jdurgin) [inline src/en/news/blog/2023/v18-2-1-reef-released/index.md:10]:
```suggestion
This is the first backport release in the Reef series, and the first with Debian packages, for Debian Bookworm. We recommend all users update to this release.
```

Comment (yuriw) [reply src/en/news/blog/2023/v18-2-1-reef-released/index.md]: done

**Review by ljflores — APPROVED** [review]

**Comment by yuriw** [comment]
jenkins test this please

**Comment by yuriw** [comment]
jenkins test this please

**Pull request merged**